stdlib/2/xmlrpclib.pyi

```diff
@@ -0,0 +1,193 @@
+    def dump_struct(self, value: str, write: Callable[[str], None], escape: Callable[[AnyStr, Callable[[AnyStr, AnyStr, AnyStr], AnyStr]], AnyStr] = ...) -> None: ...
+    def dump_datetime(self, value: str, write: Callable[[str], None]) -> None: ...
+    def dump_instance(self, value: str, write: Callable[[str], None]) -> None: ...
+
+class Unmarshaller:
+    def append(self, object: Any) -> None: ...
+    def __init__(self, use_datetime: bool = ...) -> None: ...
+    def close(self) -> tuple: ...
+    def getmethodname(self) -> Optional[str]: ...
+    def xml(self, encoding: str, standalone: bool) -> None: ...
+    def start(self, tag: str, attrs: Any) -> None: ...
+    def data(self, text: str) -> None: ...
+    def end(self, tag: str, join: Callable[[Iterable[str], str], str] = ...) -> None: ...
+    def end_dispatch(self, tag: str, data: str) -> None: ...
+    dispatch = ...  # type: Mapping[str, Callable[[Unmarshaller, str], None]]
+    def end_nil(self, data: str): ...
+    def end_boolean(self, data: str) -> None: ...
+    def end_int(self, data: str) -> None: ...
+    def end_double(self, data: str) -> None: ...
+    def end_string(self, data: str) -> None: ...
+    def end_array(self, data: str) -> None: ...
+    def end_struct(self, data: str) -> None: ...
+    def end_base64(self, data: str) -> None: ...
+    def end_dateTime(self, data: str) -> None: ...
+    def end_value(self, data: str) -> None: ...
+    def end_params(self, data: str) -> None: ...
+    def end_fault(self, data: str) -> None: ...
+    def end_methodName(self, data: str) -> None: ...
+
+class _MultiCallMethod: ...
+class MultiCallIterator: ...
+
+class MultiCall:
+    def __init__(self, server: ServerProxy) -> None: ...
+    def __getattr__(self, name: str) -> _MultiCallMethod: ...
+    def __call__(self) -> MultiCallIterator: ...
+
+def getparser(use_datetime: bool = ...) -> Tuple[Union[ExpatParser, SlowParser], Unmarshaller]: ...
+def dumps(params: Union[tuple, Fault], methodname: Optional[str] = ..., methodresponse: Optional[bool] = ..., encoding: Optional[str] = ..., allow_none: bool = ...) -> str: ...
+def loads(data: str, use_datetime: bool = ...) -> Tuple[tuple, str]: ...
+
+def gzip_encode(data: str) -> str: ...
+def gzip_decode(data: str, max_decode: int = ...) -> str: ...
+
+class GzipDecodedResponse:
+    stringio = ...  # type: StringIO
+    def __init__(self, response: HTTPResponse) -> None: ...
+    def close(self): ...
+
+class _Method:
+    def __init__(self, send: Callable[[str, tuple], Any], name: str) -> None: ...
+    def __getattr__(self, name: str) -> _Method: ...
+    def __call__(self, *args: tuple) -> Any: ...
+
+class Transport:
+    user_agent = ...  # type: str
+    accept_gzip_encoding = ...  # type: bool
+    encode_threshold = ...  # type: Optional[int]
+    def __init__(self, use_datetime: bool = ...) -> None: ...
+    def request(self, host: _hostDesc, handler, request_body: str, verbose: bool = ...) -> tuple: ...
```

Comment:
Seems `handler` is just a `str`:
https://github.com/python/cpython/blob/2.7/Lib/xmlrpclib.py#L1411
https://github.com/python/cpython/blob/2.7/Lib/httplib.py#L892